cmd/cluster/main.go

```diff
@@ -23,5 +27,16 @@ func main() {
 	c := newCluster()
 	c.AddFlags(pflag.CommandLine)
 	pflag.Parse()
-	_ = c.Run()
+
+	ctx, cancel := context.WithCancel(context.Background())
+	chCancel := make(chan os.Signal)
+	signal.Notify(chCancel, syscall.SIGINT, syscall.SIGTERM)
+	go func() {
+		fmt.Printf("Waiting\n")
+		<-chCancel
+		fmt.Printf("cancelled\n")
+		cancel()
+	}()
```

Comment:
```suggestion
    ctx, done := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
```

Reduce the amount of cruft you need?

Reply:
I'll include it in the fix for the gofmt /tableflip